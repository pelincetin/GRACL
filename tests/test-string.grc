int main() {
    String s = "Hello";
    String t = "hello";
    String empty = "";
    printi(s.stringLength());
    printi(empty.stringLength());
    if (s.stringEquals(t)) print("Equals");
    else print("Not Equals");
    if (s.stringEquals("Hello")) print("Equals");
    else print("Not Equals");
}