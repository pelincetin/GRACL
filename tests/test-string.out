5
0
Not Equals
Equals